provision/aws/rosa_delete_cluster.sh

```diff
@@ -25,9 +25,14 @@ if [ -z "$REGION" ]; then echo "Variable REGION needs to be set."; exit 1; fi
 
 cd ${SCRIPT_DIR}/../opentofu/modules/rosa/hcp
 WORKSPACE=${CLUSTER_NAME}-${REGION}
-./../../../destroy.sh ${WORKSPACE}
+./../../../destroy.sh ${WORKSPACE} &
+DESTROY_PROCESS_ID=$!
 
 LOG_DIR="${SCRIPT_DIR}/logs/${CLUSTER_NAME}"
 mkdir -p ${LOG_DIR}
 cd ${LOG_DIR}
-rosa logs uninstall --debug -c ${CLUSTER_NAME} > "$(custom_date)_delete-cluster.log"
+echo "Saving Uninstall Logs."
+rosa logs uninstall -c ${CLUSTER_NAME} --watch --debug > "$(custom_date)_delete-cluster.log" &
+LOGS_SAVING_PROC_ID=$!
+wait $DESTROY_PROCESS_ID $LOGS_SAVING_PROC_ID
```

Comment:
is there a chance for it to be stuck in this wait state for longer than intended ? should we have a timeout set ?

Reply:
@kami619 thanks for review. Please see my comment below.